zerver/lib/actions.py

```diff
@@ -367,16 +367,26 @@ def send_message_to_signup_notification_stream(
         internal_send_stream_message(sender, signup_notifications_stream, topic_name, message)
 
 
-def notify_new_user(user_profile: UserProfile) -> None:
+def notify_new_user(user_profile: UserProfile, acting_user: Optional[UserProfile] = None) -> None:
```

Comment:
Let's call this variable `referring_user`; it's not really the same as the typical `acting_user` parameter.

Reply:
I guess we should do a preparatory commit changing this name in the caller, too, so that we have `acting_user=referring_user,` in the existing call to `bulk_add_subscriptions`.